### PR TITLE
Update and fix typos in getting_started.md

### PR DESCRIPTION
**Pull request by tokcum**
Just started to explore the opportunities crev offers to a Secure Development Lifecycle and crossed some minor issues in "Getting started".

Checklist:

* [ ] `cargo +nightly fmt --all`
* [ ] Modify `CHANGELOG.md` if applicable

### CODE REVIEW

File: cargo-crev/src/doc/getting_started.md [hunk 1]
```diff
@@ -85,29 +108,61 @@ When installed `cargo-crev` can be run like this:
...
 
 As you can see, by default `cargo crev` displays the built in help. Try it and
 scan briefly over `SUBCOMMANDS` section. It should give you a good overview
 of the available functionality.
 
+## Using Subcommands
+
+A subcommand determines the action taken by `cargo-crev`. Some subcommand of `cargo-crev` offer 
+an additional level of subcommands. Some of these cascaded subcommands are provided by shortcuts, 
+such as `verify` for `crate  verify`. For specific help regarding a subcommand use the flag `-h`.
```
Comment (ffranr) [inline cargo-crev/src/doc/getting_started.md:143]:
```suggestion
such as `verify` for `crate  verify`. For specific help regarding a subcommand use the `-h` flag.
```

File: cargo-crev/src/doc/getting_started.md [hunk 2]
```diff
@@ -85,29 +108,61 @@ When installed `cargo-crev` can be run like this:
...
+
+Note: You can abbreviate most of `cargo-crev` subcommands. So you can save some keypresses with: `cargo crev c v`.
```
Comment (ffranr) [inline cargo-crev/src/doc/getting_started.md:145]:
```suggestion
Note: You can abbreviate most `cargo-crev` subcommands. For example: `cargo crev c v`.
```

File: cargo-crev/src/doc/getting_started.md [hunk 3]
```diff
@@ -85,29 +108,61 @@ When installed `cargo-crev` can be run like this:
...
+
+```
+text
```
Comment (ffranr) [inline cargo-crev/src/doc/getting_started.md:148]:
```suggestion
```text
```

File: cargo-crev/src/doc/getting_started.md [hunk 4]
```diff
@@ -143,10 +195,10 @@ On the right side `crate` and `version` indicate for which crate (in a given ver
 values in other columns are calculated and displayed for.
 
 The `status` column displays the verification status for each crate. A `pass` value
-indicates it has been reviewed by enough trusted people to consider it trustworthy.
+indicates it has been reviewed by enough trusted people (default is `1`) to consider it trustworthy.
```
Comment (ffranr) [inline cargo-crev/src/doc/getting_started.md:198]:
```suggestion
indicates that it has been reviewed by a sufficient number of trusted peers.
```

I don't think we need to provide the default value here.

File: cargo-crev/src/doc/getting_started.md [hunk 5]
```diff
@@ -284,14 +336,14 @@ comment: ""
 
 Editing the proof is modeled after editing a commit message through `git commit`.
 As you can see helpful documentation is available in the editor. Don't forget
```
Comment (ffranr) [inline cargo-crev/src/doc/getting_started.md:338]: Link on "helpful documentation" instead?

File: cargo-crev/src/doc/getting_started.md [hunk 6]
```diff
@@ -327,23 +379,19 @@ Fetching https://github.com/dpc/crev-proofs... OK
...
 
-
 Try `cargo crev crate verify` again.
 
 If you are moderately lucky, at least some of the dependencies are now passing the verification.
 
 But ultimately someone has to do the review, and at least sometimes you will have to do it yourself.
 
-Scan the output of `cargo crev crate verify` and pick a crate with low `lines` count. For your first
+Scan the output of `cargo crev crate verify` and pick a crate with low `loc` count. For your first
```
Comment (ffranr) [inline cargo-crev/src/doc/getting_started.md:392]:
```suggestion
Scan the output of `cargo crev crate verify` and pick a crate with low lines of code (`loc`) count. For your first
```

File: cargo-crev/src/doc/getting_started.md (hunk 5 above)
Comment (tokcum) [reply cargo-crev/src/doc/getting_started.md]: Sure. Done.

**Comment by dpc** [comment]
Awesome! Thanks! Much appreciated!

**Comment by tokcum** [comment]
> Awesome! Thanks! Much appreciated!

Thank you for crev. More to come. :-)

**Pull request merged**